### ISSUE TITLE
Logviewer should suggest crontask rather than requiring it

### ISSUE THREAD

**Issue by robbieaverill** [silverleague/silverstripe-logviewer#27]
The purpose of the crontask is that if you've got the module installed, you can run the clean up build task as a cron task as well. It shouldn't require it as a dependency though.

**Comment by zanderwar** [comment]
Any reason why we can't move this to `require-dev` ?

**Comment by robbieaverill** [comment]
Yeah the BuildTask implements CronTask at the moment, initially it was to save duplicating the code. We'd need to separate that and create a separate CronTask implementation which we could then suggest crontask instead using
